Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM praekeltfoundation/django-bootstrap:py3.6
```

Comment:
We're linting with Python 3.10, why not use the same version here? 3.6 is no longer supported by Python